File: PHP/DesignPattern/FM_S/ChicagoStyleCheesePizza.php
<?php
	include_once 'Pizza.php';

	/**
	* 具体的产品
	* ChicagoStylePizza封装的是关于如何制造芝加哥风味的比萨
	*/
	class ChicagoStyleCheesePizza extends Pizza {
		public function __construct() {
			$this->name = "Chicago Style Sauce and Cheese Pizza";
		}
	}

File: Java/FactoryMethod/src/com/base/ChicagoStyleCheesePizza.java
package com.base;

import com.base.Pizza;

/**
 * Created by hhq on 1/14/15.
 */
public class ChicagoStyleCheesePizza extends Pizza {
    public ChicagoStyleCheesePizza() {
        name = "Chicago Style Sauce and Cheese com.base.Pizza";
        dough = "Extra Thick crust Dough";
        sauce = "Plum Tomato Sauce";

        toppings.add("Shredded Mozzarella Cheese");
    }
    
    void cut() {
        System.out.println("Cutting the pizza into square slices");
    }
}
